### DISCUSSION TITLE
关于播放列表的2个问题

### DISCUSSION THREAD

**Discussion by Yumeyo23** [ywmoyue/biliuwp-lite#724] [bug]
### Bug 描述

作者辛苦了，谢谢！这次反馈播放列表的2个问题描述如下，后面有BUG现象GIF录制图：

1.**有概率**无法打开部分播放列表（合集），特别是只有1个视频的播放列表，但该列表在网页B站已确定可以正常打开
注：关于提示的运行时问题，我的理解是这样的：如果打开合集需要运行时，那么我的电脑没有装运行时，理论上是所有合集都打不开才对，但是我是有些合集可以打开，有些打不开，所以我认为这个跟运行时关系不大，应该是一个问题
![1](https://github.com/user-attachments/assets/1e0ddc2f-f5a2-4d9a-bd25-f01cff9d623c)

2.在开启了“右侧视频详情宽度可调整”后，开启最大化模式，点击任意播放列表，立刻调整列表宽度，**有概率出**现视频区域无法跟随变化大小，且此时视频无法正常播放。（优先解码器设置为H265）
![22](https://github.com/user-attachments/assets/68d8ea4d-450b-4374-a45b-00c2b6cb8d6a)

版本是目前最新测试版

这两个问题在我两台WIN10电脑上都可复现，但是**是概率复现**特别是问题2，如GIF所示，有时候能调，有时候不能调
我也不太敢确认这是BUG还是我的问题，但是考虑到我两台电脑可以复现，我还是反馈一下
我不是很懂软件开发，还请多多指教，如果是我的问题话，很抱歉打扰了（
谢谢！祝软件越来越好

### 操作步骤

```plain text
如上
```

### 期望结果

如上

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment]
第一个问题没看到gif中有在软件内打开描述的能打开的合集页面，不能确定是同一个页面，如果说不是从用户->合集页进入而是从视频中加载到的合集那么后者是不需要webview2运行时的

第二个问题你试试将视频画面尺寸设置为兼容

**Reply by Yumeyo23** [reply]
作者下午好，第一个问题抱歉我之前不是很懂，另一个小伙伴在二楼解释了，现在我懂了，感谢
第二个问题我刚试了一下，把画面调成兼容模式后，刚才测试了10个左右我能打开的合集，调整尺寸已经正常了
但是一旦从兼容换回默认，问题复现
这是哪方面的问题呢，算BUG吗🤣
请多多指教，谢谢😘

**Reply by ywmoyue** [reply]
https://github.com/ywmoyue/biliuwp-lite/pull/475
之前是这个PR加的兼容模式，后面可以再看看是不是把兼容模式改成默认选项

**Comment by GD-Slime** [comment]
合集目前是将两个模式合并来的，一个是season, 一个是series. 其中series因为适配原因目前是调用Webview（即浏览器套壳？）来打开的。 如果需要打开这种的话需要安装一下Webview运行时。
![image](https://github.com/user-attachments/assets/28c753b3-f80f-49e5-a7a1-fc0ba4fa2b03)
可以从这里的season和series看出是使用哪种方式打开的

**Reply by Yumeyo23** [reply]
了解了，谢谢！我也是才注意到是这两个的区别，抱歉之前不是很懂
